point.py

```diff
@@ -0,0 +1,108 @@
+import random
+import math
+
+
+class Point:
+    """
+    For whatever reason, cannot get import to work, just moved functionality
+    into this file instead.
+    """
+    def __init__(self,x = 0,y = 0,mark = ""):
+        self.x = x
+        self.y = y
+        self.mark = mark
+
+    def shift_point(self,x_move,y_move):
+        this_point = (self.x,self.y)
+        self.x += x_move
+        self.y += y_move
```

Comment:
Could you 'patch' this functionality in from the `utils` module (once imports are working)?